data/json/items/tool_armor.json

```diff
@@ -60,6 +60,44 @@
     "techniques": [ "WBLOCK_1" ],
     "magazine_well": 1
   },
+  {
+    "id": "welding_mask",
+    "type": "ARMOR",
+    "name": "welding mask",
+    "description": "A steel mask with a very dark visor, meant to protect the entire face while welding.  It also serves as good armor, but makes it hard to see.  Activate it to push it up your head.",
+    "weight": 4000,
+    "volume": "4 L",
+    "price": 1400,
+    "to_hit": -3,
+    "use_action": { "type": "transform", "msg": "You push your %s up your head.", "target": "welding_mask_raised", "menu_text": "Raise" },
+    "material": [ "steel" ],
+    "symbol": "[",
+    "color": "dark_gray",
+    "covers": [ "HEAD", "MOUTH", "EYES" ],
+    "coverage": 90,
+    "warmth": 5,
+    "encumbrance": 60,
+    "material_thickness": 3,
+    "qualities": [ [ "GLARE", 3 ] ],
+    "flags": [ "SUN_GLASSES", "FLASH_PROTECTION" ]
+  },
+  {
+    "id": "welding_mask_raised",
+    "copy-from": "welding_mask",
+    "type": "ARMOR",
+    "name": "welding mask (raised)",
+    "name_plural": "welding masks (raised)",
+    "description": "A steel mask with a very dark visor, meant to protect the entire face while welding.  It also serves as good armor, but makes it hard to see.  Activate it to lower it over your face.",
+    "use_action": {
+      "type": "transform",
+      "msg": "You lower your welding mask over your face.",
+      "target": "welding_mask",
+      "menu_text": "Lower"
+    },
+    "covers": [ "HEAD" ],
+    "delete": { "flags": [ "SUN_GLASSES", "FLASH_PROTECTION" ] },
+    "coverage": 80
```

Comment:
A welding mask with the visor raised is way less than 80% coverage, more like 30%